### PR TITLE
2 get api topics

### PR DESCRIPTION
**Pull request by yvevolk**
Added get-topics endpoint.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('./controllers/controllers.js')
+
+app.get('/api/topics', getTopics);
+
+app.all('/api/*', function (req, res, next) {
+    res.status(404).send({message: 'error, invalid endpoint'})
+})
+
```
Comment (yalicee) [inline app.js:10]: Please add a 500 error handler as well

File: controllers/controllers.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require('../models/models.js')
+
+exports.getTopics = (req, res, next) => {
+    fetchTopics().then((result) => {
+        res.status(200).send({topics: result})
+    })
```
Comment (yalicee) [inline controllers/controllers.js:6]: You could add a .catch block to improve your code

**Review by yalicee — APPROVED** [review]
Have a look at my comments but otherwise happy for you to merge after you have read them

**Comment by yvevolk** [comment]
> Have a look at my comments but otherwise happy for you to merge after you have read them

Have done, will add in subsequent PRs.

**Pull request merged**